pyproject.toml

```diff
@@ -50,15 +49,20 @@ tnflow = [
     "numpy"
 ]
 tests = [
-    "pytest"
+    "pytest>=7"
+]
+coverage = [
+    "mqt.ddsim[tests]",
+    "pytest-cov[toml]"
 ]
 docs = [
     "sphinx>=5",
     "sphinx-rtd-theme",
     "sphinx-rtd-dark-mode",
     "sphinxcontrib-bibtex>=2.4.2",
     "sphinx-copybutton",
-    "sphinxext-opengraph"
+    "sphinxext-opengraph",
+    "breathe"
 ]
```

Comment:
It's common best practice to also introduce a `dev` extra here that subsumes all of the available extras. You can find an example for that in QCEC or QMAP.

Reply:
Will be added in the next commit